test/suites/tracing-tests/test-trace-evm-foreign-asset-registration.ts

```diff
@@ -0,0 +1,42 @@
+import { customDevRpcRequest, describeSuite, expect } from "@moonwall/cli";
+import {
+  ARBITRARY_ASSET_ID,
+  registerForeignAsset,
+  RELAY_SOURCE_LOCATION_V4,
+  relayAssetMetadata,
+} from "../../helpers";
+
+describeSuite({
+  id: "T17",
+  title: "Trace ERC20 Foreign asset creation",
+  foundationMethods: "dev",
+  testCases: ({ context, it }) => {
+    it({
+      id: "T01",
+      title: "Ensure native ERC20 foreign asset creation is traceable",
+      timeout: 50000,
+      test: async function () {
+        const assetLocation = RELAY_SOURCE_LOCATION_V4;
+        const assetId = ARBITRARY_ASSET_ID;
+
+        // Register the asset
+        await registerForeignAsset(context, assetId, assetLocation, relayAssetMetadata);
```

Comment:
`registerForeignAsset` does not use XCM from a SiblingPara, but Sudo. Do all `createForeignAsset` calls go through XCM Transact -> Ethereum XCM?

Reply:
Yes, they use the same evm runner as the one for XCM. And will be callable from parachains in the PR you and Tarek are currently working on.